exec/RenderHtml.hs

```diff
@@ -75,32 +75,38 @@ pages f =
     ]
 
 
+-- | We write 'ToHtml' for pages that contain no forms, and 'FormPageView' for pages that do.  In
+-- this module, we need to render both into html for viewing only, and 'ToHtml'' is introduced for
```

Comment:
Haddock accepts the @ToHtml'@ form as well and it may look better here. Haddock doesn't generate links in that case, which is actually good, because we refer to the definition immediately below. Also one link per entity in one haddocks is usually enough, so the @foo@ form could be used below as well.